apparmor.d/profiles-s-z/waybar

```diff
@@ -0,0 +1,45 @@
+# apparmor.d - Full set of apparmor profiles
+# Copyright (C) 2024 odomingao
+# SPDX-License-Identifier: GPL-2.0-only
+
+abi <abi/3.0>,
+
+include <tunables/global>
+
+@{exec_path} = @{bin}/waybar
+
+profile waybar /{,usr/}{,s}bin/waybar flags=(attach_disconnected) {
+  include <abstractions/audio>
+  include <abstractions/base>
```

Comment:
base must come before audio. (nitpick but, let's respect the general guideline)

Reply:
Got it, but the documentation only says that the order of the rules should be alphabetical, unless missed it.

Reply:
Yes, but the base abstraction is special.